integration_tests/test_dbapi.py

```diff
@@ -368,15 +360,15 @@ def test_transaction_multiple(trino_connection_with_transaction):
     assert len(rows1) == 1000
     assert len(rows2) == 1000
 
+
 def test_invalid_query_throws_correct_error(trino_connection):
-    """
-    tests that an invalid query raises the correct exception
+    """Tests that an invalid query raises the correct exception
     """
     cur = trino_connection.cursor()
     with pytest.raises(TrinoQueryError):
         cur.execute(
             """
-            select * FRMO foo WHERE x = ?; 
+            select * FRMO foo WHERE x = ?;
```

Comment:
While we are at it, can you make SQL queries use uppercase keywords (or at-least be consistent within a single query) if it's not too much of an effort?

It should be a separate commit though.